docs/core/reference/corectl.md

```diff
@@ -141,3 +141,45 @@ other configuration will be deleted.
 ```
 corectl reset
 ```
+
+### `grant`
+
+Command grant grants access to a policy
+for the described credentials.
+
+```
+corectl grant [-cn name] [-ou name] [-t token] [policy]"
+```
+
+Flag -cn grants access to any valid X.509 certificate
```

Comment:
It'd be nice to match the flag formatting of previous comments, or update those to match this.